src/pyOpenMS/addons/streampos.pyx

```diff
@@ -1,8 +1,8 @@
 
 
-    def __long__(self):
+    def __int__(self):
         # <long> calls c++ conversion streampos -> c++ long value
-        # long(..) converts this value to a Python long.
+        # int(..) converts this value to a Python long.
         # value. else this method may return an int or long, depending on the magnitude of
         # the value !
-        return long(<long>(deref(self.inst.get())))
+        return int(<long>(deref(self.inst.get())))
```

Comment:
we should think about this. is cast to long necessary?

Reply:
maybe the whole addon is not needed anymore?

Reply:
I would guess that it's not, but my understanding of why we had it is cursory at best. Was this to handle different size integer values?

Reply:
we should check where it is used and add a test to see if it fails if this addon is removed. I also have the feeling it is not necessary anymore now that python long is completely gone.

Reply:
As far as I can tell it's not used explicitly anywhere. My concern was that it gets used somewhere implicitly.

Reply:
It is used here:
https://github.com/OpenMS/OpenMS/blob/16e128abb4e59366c1555744b0829079ee90108b/src/pyOpenMS/tests/unittests/test000.py#L5175